lib/logger/utils.ts

```diff
@@ -32,3 +35,56 @@ export class ErrorStream extends Stream {
     return this._errors;
   }
 }
+
+function sanitizeObject(obj, seen = new WeakMap()) {
```

Comment:
```suggestion
function sanitizeObject(obj: Record<string, any>, seen = new WeakMap()): Record<string, any> {
```

Maybe additional formating required.